IlmBase/IlmThread/IlmThreadPool.cpp

```diff
@@ -536,34 +540,34 @@ TaskGroup::Data::removeTask ()
 
     // Further update:
     //
-    // we could remove this if it is a new enough glibc, however 
+    // we could remove this if it is a new enough glibc, however
```

Comment:
If there's a desire to clean up white space we always appreciate that such cleaning be part of a separate PR that only does cleaning. It's difficult to be sure a review is thorough when there are many of these to be checked.

Reply:
I only used the Codeblocs editor which did it myself.

Reply:
I noticed there are a lot of questions about this in the CodeBlocks forums, maybe there's an answer there about how to turn off that feature. It's nice to have that feature when you are working in your codebase, but when working in open source it makes things more confusing for your collaborators.